world-server/src/world/chat/ChatHandlers.ts

```diff
@@ -0,0 +1,46 @@
+import { Messages, Router, Logger } from 'world-core';
+import { BroadcastMessage } from '../../network';
+import { ChatMessage } from './ChatMessages';
+import { Player } from '../player';
+
+export class ChatHandler implements Messages.MsgHandler {
+    private players: Player[];
+
+    constructor(players: Player[]) {
```

Comment:
This parameter is copy & pasted and not actually used anywhere. I believe it should be removed.